src/Protocol/Protocol.h

```diff
@@ -131,6 +131,8 @@ class cProtocol
 	virtual void SendSpawnVehicle               (const cEntity & a_Vehicle, char a_VehicleType, char a_VehicleSubType) = 0;
 	virtual void SendStatistics                 (const cStatManager & a_Manager) = 0;
 	virtual void SendTabCompletionResults       (const AStringVector & a_Results) = 0;
+	virtual void SendTeam                       (const cTeam & a_Team, Byte a_Mode) = 0;
```

Comment:
Use the `eProtocolAction` enum here as well, only convert in the last moment before sending.